types/node/index.d.ts

```diff
@@ -2834,7 +2834,7 @@ declare module "fs" {
      * @param fileName
      * @param options An object with optional {encoding} and {flag} properties.  If {encoding} is specified, readFileSync returns a string; otherwise it returns a Buffer.
      */
-    export function readFileSync(filename: string, options: { encoding: string; flag?: string; }): string;
+    export function readFileSync(filename: string, options: { encoding?: string | null; flag?: string; }): string;
```

Comment:
Good catch. I've updated the definitions to account for this. The problem is that I'm always passing in `encoding`, but sometimes the value is `null`, so I need that to be accounted for in the typings.